### PR TITLE
Refs #20461 -- Dropped runtests' message about parallel processes.

### PR DESCRIPTION
**Pull request by aaugustin**
(EDITED)

The actual number of processes isn't known until `test_runner.run_tests()` loads tests and determines how many TestCases will run.

### CODE REVIEW

File: tests/runtests.py [hunk 1]
```diff
@@ -253,6 +250,11 @@ def django_tests(verbosity, interactive, failfast, keepdb, reverse,
     state = setup(verbosity, test_labels, parallel)
     extra_tests = []
 
+    parallel = actual_test_processes(parallel)
+    if verbosity >= 1:
+        print("Testing with %d parallel processes..." % parallel)
+
```
Comment (timgraham) [inline tests/runtests.py]: LGTM besides extra newline that fails flake8.

Comment (aaugustin) [inline tests/runtests.py]: I'll merge later. Enough problems for today.

**Comment by aaugustin** [comment]
It's good that I didn't merge my first attempt because it doesn't work....

I think we have to remove the message. The commit message explains why.

File: tests/runtests.py [hunk 2]
```diff
@@ -265,7 +262,7 @@ def django_tests(verbosity, interactive, failfast, keepdb, reverse,
         keepdb=keepdb,
         reverse=reverse,
         debug_sql=debug_sql,
-        parallel=actual_test_processes(parallel),
+        parallel=parallel,
```
Comment (timgraham) [inline tests/runtests.py]: I think this change is inadvertent as now `actual_test_processes()` is unused.

**Comment by timgraham** [comment]
Alternatively the message could say "with up to X processes." (knowing that the number could be reduced if there are less test cases). I don't have a strong feeling either way.

**Comment by aaugustin** [comment]
"up to X processes" is smart. Let me try again :-)

**Comment by aaugustin** [comment]
I just pushed a new version. I did some manual testing. It appears to work.

**Comment by timgraham** [comment]
Looks good.
